rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -1659,4 +1660,11 @@ public ClearingRequest updateCRSize(ClearingRequest clearingRequest, Project pro
     public boolean isWriteActionAllowed(Object object, User user) {
         return makePermission(object, user).isActionAllowed(RequestedAction.WRITE);
     }
+
+    public void isSecurityUser(User user) {
```

Comment:
Please use a more helpful name like `throwIfSecurityUser()` as it currently feels like just a validator which returns `true` if the user is `SECURITY_USER`.